src/main/java/org/jenkinsci/plugins/workflow/cps/DSL.java

```diff
@@ -414,32 +443,27 @@ protected Object invokeDescribable(String symbol, Object _args) {
                 ud = new UninstantiatedDescribable(symbol, null, dargs);
                 margs.put(p.getName(),ud);
 
-                return invokeStep(metaStep, symbol, new NamedArgsAndClosure(margs, args.body));
+                return invokeStep(metaStep, symbol, new NamedArgsAndClosure(margs, args.body, null));
             } catch (Exception e) {
                 throw new IllegalArgumentException("Failed to prepare "+symbol+" step",e);
             }
         }
     }
 
-    private void reportAmbiguousStepInvocation(CpsStepContext context, StepDescriptor d) {
+    private void reportAmbiguousStepInvocation(CpsStepContext context, StepDescriptor d, @Nullable TaskListener listener) {
         Exception e = null;
```

Comment:
Unused?

Reply:
There is one call to this method here: https://github.com/jenkinsci/workflow-cps-plugin/blob/ade619f5aa4f3d22811164614e0460a95da6df49/src/main/java/org/jenkinsci/plugins/workflow/cps/DSL.java#L287

Reply:
Sorry, I meant `Exception e = null;` is unused if you make the change I suggested below.

Reply:
unused exception  removed  in 79d255a8427bc41a575bb6f808b355dfaadecb3e